src/core/util/posix/tmpfile.cc

```diff
@@ -48,14 +48,15 @@ FILE* gpr_tmpfile(const char* prefix, char** tmp_filename) {
 
   fd = mkstemp(filename_template);
   if (fd == -1) {
-    gpr_log(GPR_ERROR, "mkstemp failed for filename_template %s with error %s.",
-            filename_template, grpc_core::StrError(errno).c_str());
+    LOG(ERROR) << "mkstemp failed for filename_template " << filename_template
+               << " with error " << grpc_core::StrError(errno).c_str();
     goto end;
   }
   result = fdopen(fd, "w+");
   if (result == nullptr) {
-    gpr_log(GPR_ERROR, "Could not open file %s from fd %d (error = %s).",
-            filename_template, fd, grpc_core::StrError(errno).c_str());
+    LOG(ERROR) << "Could not open file " << filename_template << " from fd "
+               << fd << " (error = " << grpc_core::StrError(errno).c_str()
```

Comment:
Is c_str() still needed here? Same as above.

Reply:
grpc_core::StrError returns std::string which is supported by ABSL << overloaded operator.

Reply:
Fixed